### ISSUE TITLE
#2 Bit App warning on High Sierra

### ISSUE THREAD

**Issue by jeffmc** [tedchoward/Frontier#7]
32 Bit apps will no longer run on newer OSX/macOS versions, as the 32 bit RunTime is deprecated. Getting the "not optimized" warning on launch. You probably know this, but login issue.

**Comment by tedchoward** [comment]
Yeah, the main problem here is that the Frontier kernel uses a lot of Carbon APIs, which are not available for 64-bit builds.

**Comment by tedchoward** [comment]
Also, it should still run. I'm running it right now on macOS 10.13.4.

The warning is for future release of macOS. The writing is on the wall.
